types/file-exists/file-exists-tests.ts

```diff
@@ -5,11 +5,17 @@ fileExists('/index.html', (err, exists) => {
     exists; // $ExpectType boolean
 });
 
-fileExists('/index.html', {root: ''}, (err, exists) => {
+fileExists('/index.html', { root: '' }, (err, exists) => {
```

Comment:
```suggestion
// $ExpectType void
fileExists('/index.html', { root: '' }, (err, exists) => {
```

And the same for the first test above.